pufferlib/ocean/trash_pickup/trash_pickup.c

```diff
@@ -0,0 +1,74 @@
+#include <time.h>
+#include "trash_pickup.h"
+
+#define INDEX_C(env, x, y) ((y) * (env).grid_size + (x))
```

Comment:
Just make a function. Don't use macros for this... or really anything tbh

Reply:
Changed (to be pushed).